test/integration/test_htmlcontains.py

```diff
@@ -0,0 +1,44 @@
+from ftw import ruleset, testrunner, http, errors
+import pytest
+import re
+import random
+import threading
+
+def test_logcontains(ruleset, test):
+    runner = testrunner.TestRunner()
+    for stage in test.stages:
+        runner.run_stage(stage)
+
+# Should return a test error because its searching before response
```

Comment:
For these tests, I think if you add a `"""foo"""` comment underneath the function definition then the tests should display it. I could be wrong tho